### PR TITLE
issue4083 change prefix meaning

### LINKED ISSUE #4083: PSWI - z/OSMF Workflows: ZWEV2.SZWEEXEC no created
When running `zwe init mvs` in V3 I get the following messages:

```
>> Initialize Zowe custom data sets

Create data sets if they do not exist
Creating dkelosky.ZWEV2.CUST.PARMLIB
Creating dkelosky.ZWEV2.CUST.JCLLIB
Creating dkelosky.ZWEV2.SZWEAUTH
Creating dkelosky.ZWEV2.CUST.ZWESAPL

Copy dkelosky.ZWEV2.SZWESAMP(ZWESIP00) to dkelosky.ZWEV2.CUST.PARMLIB(ZWESIP00)
  * Exit code: 12
  * Output:
    exec 'dkelosky.ZWEV2.SZWEEXEC(ZWEMCOPY)' 'dkelosky.ZWEV2.SZWESAMP(ZWESIP00) dkelosky.ZWEV2.CUST.PARMLIB(ZWESIP00)'
    DATA SET DKELOSKY.ZWEV2.SZWEEXEC NOT IN CATALOG OR CATALOG CAN NOT BE ACCESSED
Error ZWEL0111E: Command aborts with error.
bash-4.3$ zwe init mvs -c '/u/users/dkelosky/zowe-smpe/zowe.yaml' --allow-overwrite
```

At what point should `DKELOSKY.ZWEV2.SZWEEXEC` be created since all of the other `DKELOSKY.ZWEV2.` data sets are created during `zwe init mvs` ?

### PR DESCRIPTION
**Pull request by pavelzlatnik**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [X] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [X] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes #4083 

#### Changes proposed in this PR
Based on issue description and proposed solutions, changed "PREFIX" variable description in the workflow, and also updated variable description in example.
- 

#### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

File: workflows/files/ZWECONF.xml [hunk 1]
```diff
@@ -12,9 +12,9 @@
     </workflowInfo>
     <!--variables-->
     <variable name="zowe_setup_dataset_prefix" scope="instance" visibility="public">
-        <label>Zowe setup MVS prefix</label>
-        <abstract>Where Zowe MVS data sets will be installed</abstract>
-        <description>Where Zowe MVS data sets will be installed</description>
+        <label>Prefix of existing Zowe Target libraries</label>
+        <abstract>Prefix of existing Target libraries created during PSWI or SMP/E</abstract>
+        <description>Prefix of existing Target libraries created during PSWI or SMP/E installation phase</description>
```
Comment (muzikovam) [inline workflows/files/ZWECONF.xml]: Add here that it is HLQ of SZWEEXEC, please.

Comment (pavelzlatnik) [reply workflows/files/ZWECONF.xml]: updated

**Review by muzikovam — APPROVED** [review]

File: workflows/files/ZWECONF.xml [hunk 2]
```diff
@@ -12,9 +12,9 @@
     </workflowInfo>
     <!--variables-->
     <variable name="zowe_setup_dataset_prefix" scope="instance" visibility="public">
-        <label>Prefix of existing Zowe Target libraries</label>
-        <abstract>Prefix of existing Target libraries created during PSWI or SMP/E</abstract>
-        <description>Prefix of existing Target libraries created during PSWI or SMP/E installation phase</description>
+        <label>Prefix of existing Zowe SZWEEXEC Target libraries</label>
+        <abstract>Prefix (HLQ) of existing SZWEEXEC Target libraries created during PSWI or SMP/E</abstract>
```
Comment (MarkAckert) [inline workflows/files/ZWECONF.xml]: We should remove HLQ from these descriptions and retain `Prefix`. HLQ has special meaning as the first 8-character qualifier of a dataset, while prefix can include multiple qualifiers.

Comment (pavelzlatnik) [reply workflows/files/ZWECONF.xml]: agree

**Review by MarkAckert — APPROVED** [review]

**Pull request merged**